workitem/workitem_repository.go

```diff
@@ -538,7 +538,7 @@ func (r *GormWorkItemRepository) listItemsFromDB(ctx context.Context, criteria c
 		}
 		db = db.Limit(*limit)
 	}
-	db = db.Select("count(*) over () as cnt2 , *")
+	db = db.Select("count(*) over () as cnt2 , *").Order("execution_order")
```

Comment:
@DhritiShikhar That's not what I mean. 

In a list of 100 work items, where they have execution_order from 1 to 100. And I'm getting a List of these work items with a page size of 20. 

What is the execution_order values on page 1 ?